source/sxl_traffic_light_controller.rst

```diff
@@ -338,7 +338,7 @@ Status
    Traffic Light Controller  `S0029`_        Forced input status
    Traffic Light Controller  `S0030`_        Forced output status
    Traffic Light Controller  `S0031`_        Trigger level sensitivity for loop detector
-   Traffic Light Controller  `S0091`_        Operator logged in/out OP-panel
```

Comment:
Why does this PR remove S0091?

Reply:
that's a mistake. i probably intented to copy it. will fix

Reply:
fixed